cpp/oneapi/dal/algo/decision_forest/common.hpp

```diff
@@ -484,9 +551,28 @@ class model : public base {
         return get_class_count_impl();
     }
 
+    /// Performs Depth First Traversal of i-th tree
+    /// @tparam[in] tree_idx    Index of the tree to traverse
+    /// @tparam[in] op          This functor gets notified when tree nodes are visited
```

Comment:
It makes sense to describe expectations about that functor right in the doxygen:
- What operator should be overloaded
- Parameters that overloaded operator accepts

Reply:
added info regarding operators